### ISSUE TITLE
Support for multi-select fields in Zoho CRM

### ISSUE THREAD

**Issue by dmatthams** [verbb/formie#554]
**What are you trying to do?**
A description of what you want to happen.

We've got a form with a multi-select dropdown (also tried checkboxes) and when attempting to submit them to a Zoho multi-select field I get the error `"expected_data_type":"jsonarray"`. The data seems to be submitted as a comma separated string whilst Zoho expects a json array.

**What's your proposed solution?**
A description of how you think this could be solved, including any alternatives that you considered.

Submit multi-select dropdowns and checkboxes as a json array when sending to Zoho CRM

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by dmatthams** [comment]
Amazing, thanks so much – will give it a test this week.

**Comment by dmatthams** [comment]
Whoops didn't mean to close that before.

Just tested and works as expected – I did have to 'refresh integration' before it did.

Thanks for fixing so quickly :-)

**Comment by engram-design** [comment]
All good! Just like to keep it open to remind me to note when a new release has been tagged. And yep, you'll need to refresh the integration, which updates the cache of field-mapping settings.

**Comment by engram-design** [comment]
Fixed in [1.4.15](https://github.com/verbb/formie/releases/tag/1.4.15)
